apps/prairielearn/src/pages/publicAssessments/publicAssessments.ts

```diff
@@ -0,0 +1,45 @@
+import * as express from 'express';
+import asyncHandler from 'express-async-handler';
+
+import * as error from '@prairielearn/error';
+
+import { selectAssessments, selectCourseInstanceIsPublic } from '../../models/course-instance.js';
+import { selectCourseInstanceById } from '../../models/course-instances.js';
+import { selectCourseById } from '../../models/course.js';
+
+import { AssessmentRowSchema, PublicAssessments } from './publicAssessments.html.js';
+
+const router = express.Router({ mergeParams: true });
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    const course_instance_id = req.params.course_instance_id;
+    res.locals.course_instance = await selectCourseInstanceById(course_instance_id);
+    res.locals.course = await selectCourseById(res.locals.course_instance.course_id);
+    res.locals.urlPrefix = '/pl';
```

Comment:
Is this really necessary? I'm pretty sure this is already set.

Reply:
talking just about the urlPrefix right? ya, I think that's right.

Reply:
fixed in dd2e51b894